diff --git a/mirafetch-0.1.0/Cargo.toml b/mirafetch-0.1.0/Cargo.toml
index 04c22c8..fd0fd95 100644
--- a/mirafetch-0.1.0/Cargo.toml
+++ b/mirafetch-0.1.0/Cargo.toml
@@ -39,21 +39,6 @@ serde_with="3.4.0"
 clap = { version = "4.5.4", features = ["derive"] }
 smallvec = "1.13.2"
 
-[target.'cfg(windows)'.dependencies]
-winreg="0.50.0"
-wmi={default-features=false, version="0.13.1"}
-windows={version="*", features=[
-    "Win32_System_SystemInformation",
-    "Win32_Networking_WinSock",
-    "Win32_NetworkManagement_IpHelper",
-    "Win32_NetworkManagement_Ndis",
-], default-features=false}
-winsafe={version="0.0.17", features=["dwm"], default-features=false}
-
-[target.'cfg(target_os = "macos")'.dependencies]
-sysctl="0.5.4"
-plist="1.5.0"
-
 
 [target.'cfg(target_os = "linux")'.dependencies]
 platform-info="2.0.1"
